contiki/platform/linux/parse_config.h

```diff
@@ -11,6 +11,10 @@
 
 #define DATA_DIR INSTALL_LOCALSTATEDIR "/lib/" PACKAGE_TARNAME "/"
 
+// Time in seconds to resend the serial API to the controller
+#define MIN_TIME_RESEND_SERIAL 20 
+#define MAX_TIME_RESEND_SERIAL 80 
+#define DEFAULT_TIME_RESEND_SERIAL 50 
```

Comment:
If we want to preserve the original behavior should it it be set to 0?

hints about the values would help too.